bat_vis1.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "Bat_vis1"
+author: "Heather S. Davies"
+date: "April 11, 2018"
+output: html_document
+---
+

```

Comment:
Here is a comment which literally has no substance, but should still demonstrate the comment feature...